### ISSUE TITLE
Inline math that is too wide causes design to scroll on mobile

### ISSUE THREAD

**Issue by bfirsh** [arxiv-vanity/engrafo#98]
The fix isn't trivial – setting `overflow-x` on `<p>` makes it scroll unnecessarily sometimes and I'm not sure why. Needs playing around with.

**Comment by bfirsh** [comment]
I think the correct solution is to get mathjax to line-break. The complicated bit is we run mathjax server-side so it doesn't know how wide the page is.

Some ideas:

1. We could have a little script that runs client-side to split up rendered mathjax that is wider than the screen. It probably wouldn't split it in the correct place, but an incorrectly split equation is far better than it running off the edge of the screen.
2. Perhaps we could re-run mathjax client-side to do line breaking (but then why are we even running it server-side?!)

Either way, probably best to wait for #351.

@pkra any ideas? :)

**Comment by pkra** [comment]
Yeah, this can't really be solved by CSS. So I don't have much more to say than to file https://github.com/mathjax/mathjax-v3/issues/173.
